lib/web_ui/dev/goldens.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Please use the exact same license header as the rest of the files.

Reply:
Done